data/huion-kamvas-pro-24.tablet

```diff
@@ -0,0 +1,58 @@
+# Huion
+# Kamvas Pro 24
+# GT-240
+#
+# sysinfo.0LZKOwtz0e.tar.gz
+# https://github.com/linuxwacom/wacom-hid-descriptors/issues/351
+# Button Map:
+# (A=1, B=2, C=3, ...)
+#
+#    *-----------------------*
+#  A |                       | K
+#  B |                       | L
+#  C |                       | M
+#  D |                       | N
+#  E |                       | O
+#    |        DISPLAY        |
+#  F |                       | P
+#  G |                       | Q
+#  H |                       | R
+#  I |                       | S
+#  J |                       | T
+#    *-----------------------*
+#
+# Touch Strip Map:
+# (a=strip min, A=strip max)
+# Touch Strips are mirrored and do not map seperately
+#
+#    *-----------------------*
+#    |                       |
+#  A |                       | A
+#  a |        DISPLAY        | a
+#    |                       |
+#    *-----------------------*
+#
+
+[Device]
+Name=Huion Kamvas Pro 24
+ModelName=GT-240
+Class=Cintiq
+DeviceMatch=usb|256c|006e|Tablet Monitor Pen|HUION_M184;usb|256c|006e|Tablet Monitor Pad|HUION_M184;usb|256c|006e|Tablet Monitor Touch Strip|HUION_M184;
```

Comment:
These different device names may be merged, see [this PR draft](https://gitlab.freedesktop.org/libinput/libinput/-/merge_requests/989)